glide.lock

```diff
@@ -1,13 +1,22 @@
-hash: 6b127577a871ff8e66cd84563cd19e9b2c339a81c9e27adc37808dab3c8a1d5a
-updated: 2016-11-14T09:22:05.45573152-08:00
+hash: 69df4410f726670e69ae64ae30dbd2b1c0c5594dda5ca732c15266f852602dfa
```

Comment:
apparently `glide update` for a single package is already supported: https://github.com/Masterminds/glide/issues/92#issuecomment-147029937

I couldn't verify it today because all our dependencies are up to date, but I will try next time I explicitly need to update a single dependency.  If it doesn't work for me then I'll update that ticket with my findings.